src/node-lib/curriculum-api-2023/queries/pupilLessonOverview/pupilLessonOverview.query.ts

```diff
@@ -18,6 +18,10 @@ export const pupilLessonOverviewQuery =
       throw new OakError({ code: "curriculum-api/not-found" });
     }
 
+    if (lesson.isSensitive === true) {
+      throw new OakError({ code: "curriculum-api/not-found" });
+    }
+
```

Comment:
```suggestion
    if (lesson.isSensitive) {
      throw new OakError({ code: "curriculum-api/not-found" });
    }

```

🏄🏻 the equality check is redundant with a boolean